.eslintrc

```diff
@@ -193,7 +187,12 @@
 		],
 		"spaced-comment": [
 			2,
-			"always"
+			"always",
```

Comment:
ESlint somehow started complaining abot Doc comments, because `/*!` is not complying with a rule that requires a space after opening a comment: it should be `/* `. With this change ESLint accepts the `!` char too.